app/components/Settings/EncryptPanel/index.js

```diff
@@ -1 +1,28 @@
-export { default } from './EncryptPanel'
+// @flow
+
```

Comment:
Mostly just a nitpick but can we remove this empty line here (no line break needed after flow annotation)